index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mint Signup Page</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    <!-- Header -->
+    <header>
+        <div class="top-header">
+            <div class="right question">
+                <p>Already have an intuit account?</p>
+                <p><a href="#">I forgot my User ID or Password</a></p>
+            </div>
+            <div class="right signin">
+                <button type="button">
+                    <img src="./images/lock.png" alt="sign in"> 
+                    <span>Sign In</span>
+                </button>
+            </div>
+        </div>
+        <div class="logo center-align">
+            <img src="./images/logo.png" alt="intuit logo">
+        </div>
```

Comment:
- [ ] [REQUIRED] the Intuit products should be in images and text, not a screenshot, please fix that